libraries/chain/db_market.cpp

```diff
@@ -93,6 +92,78 @@ void database::globally_settle_asset( const asset_object& mia, const price& sett
 
 } FC_CAPTURE_AND_RETHROW( (mia)(settlement_price) ) }
 
+void database::revive_bitasset( const asset_object& bitasset, const account_id_type& shorter )
+{ try {
+   FC_ASSERT( bitasset.is_market_issued() );
+   const asset_bitasset_data_object& bad = bitasset.bitasset_data(*this);
+   FC_ASSERT( bad.has_settlement() );
+   const asset_dynamic_data_object& bdd = bitasset.dynamic_asset_data_id(*this);
+   FC_ASSERT( shorter != account_id_type() );
```

Comment:
Will remove the shorter parameter and the assertion. It's a leftover from an earlier version. revive_bitasset is only used for auto-revival after the feed has recovered, and it always creates a call_order that belongs to the issuer.